CAM Platform/api-providers.md

```diff
@@ -3370,56 +1408,72 @@ ElasticBox-Release: 4.0
-      "created":"2014-03-28 12:18:02.455710",
-      "text":"Provider 'MyAmazon' synchronized",
-      "level":40,
-      "updated":"2014-03-28 12:18:02.456441",
-      "id":"6303cd2f-3c73-4b8e-9aa3-12cdacc15093",
-      "schema":"http://elasticbox.net/schemas/provider-log"
-   },
-   {
-      "workspace":"--workspace name--",
-      "provider_id":"8c501fe3-54d7-49eb-b5d3-05016becabe3",
-      "created":"2014-03-26 14:03:41.863572",
-      "text":"Provider 'MyAmazon' synchronized",
-      "level":40,
-      "updated":"2014-03-26 14:03:41.864242",
-      "id":"3fa58165-2af8-47eb-99e9-98f68e76c9b2",
-      "schema":"http://elasticbox.net/schemas/provider-log"
-   }
-]
+ [
+    {
+       provider_id: "8132cc1d-38b0-4867-b7a3-1af7b83d4792",
+       level: 40,
+       text: "Amazon Web Services Management Console was accessed",
+       created: "2018-10-22 07:30:47.808486",
+       updated: "2018-10-22 07:30:47.808621",
+       role: "arn:aws:iam::636588615608:role/ElasticBox_Access",
+       workspace: "thomasbroadwell",
+       request_id: "e09360e1-7738-4cd9-91f2-cdd3c9f83a91",
+       operation: "console_access",
+       id: "231373e4-ab8f-4c28-8ba6-d91f331fae2c",
+       schema: "http://elasticbox.net/schemas/provider-log"
+    },
+    {
+       provider_id: "8132cc1d-38b0-4867-b7a3-1af7b83d4792",
+       level: 40,
+       text: "Image ami-2051294a in location us-east-1 added successfully.",
+       created: "2018-10-15 19:10:57.765608",
+       updated: "2018-10-15 19:10:57.766848",
+       workspace: "msa_service_account1",
+       request_id: "3ee09e46-1655-4e09-a500-0ccf83161eec",
+       operation: "sync",
+       id: "b79820d6-64c4-4103-ac0b-97cdfe672893",
+       schema: "http://elasticbox.net/schemas/provider-log"
+    }
+ ]
 ```
 
-**GET /services/providers/{provider_id}/unregisted-instances**
+## GET /services/providers/{provider_id}/unregisted-instances
 
 Retrieves a list of unregistered instances found in a provider when you give the provider ID.
 
-**Normal Response Codes**
+### URL
+#### Structure
+    [GET] /services/providers/{provider_id}/unregisted-instances
 
-* 200
-
-**Error Response Codes**
-
-* Forbidden (403)
-* Not Found (404)
-
-**Request**
+#### Example
+    GET https://cam.ctl.io/services/providers/338f38dc-e667-47e0-9026-b253138f109e/unregisted-instances
 
+### Request
+#### Headers
 ```
-Headers:
-
 Content-Type: application/json
 Authorization: Bearer your_json_web_token
 ElasticBox-Release: 4.0
 ```
 
-**Response parameters**
+#### URI Parameters
+| Parameter | Type | Description | Req. |
+| --- | --- | --- | --- |
+| Provider id | string | The unique id that given to each provider | Yes |
 
+### Response
+
+#### Normal Response Codes
+- **200** Accepted
+
+#### Common Error Response Codes
+- 403: Forbidden - User doesn’t belong to the organization
+- 404: Not Found
+
+#### Response Parameters
 |Parameter | Type | Description |
 |----------|------|-------------|
 |profile | object | Instance profile. |
```

Comment:
Please review the `profile`, the profile is too AWS specific and there are other clouds that support this. Maybe leave it as `profile`, stating that it's a sub-document, and point to the instances API (which should have a description of this in more detail).